popup.js

```diff
@@ -0,0 +1,84 @@
+// popup window work starts from here
+
+import jobTags from './job-tags.js';
+import techTags from './tech-tags.js';
+
+const openModal = (project) => {
+  const projects = document.createElement('div');
+  projects.className = 'popup';
+
+  const articlePopup = document.createElement('article');
+  articlePopup.className = 'article';
+
+  const divPopup = document.createElement('div');
+  divPopup.className = 'popup-div';
+  articlePopup.append(divPopup);
+
+  const h2 = document.createElement('h2');
+  h2.innerText = project.name;
+  divPopup.append(h2);
+
```

Comment:
- [ ] Please, make sure your popup stick to the Figma popup template. Actually, your popup has the following issues:

1. Your popup window is not responsive. On mobile your cards are orange and the popup doesn't show as required. Kindly, fix that issue.
2. Ensure that the popup doesn't fill the whole screen. Following the template design, it has a blue blur `background-color` and is centred in its container.
3. All your popup content should be displayed on the screen. On my PC I can't see any buttons and cannot scroll the way to check them. Please fix it.

Note: Your popup should be similar to the one provided in the Figma template to get approved.

Images:
**Your popup on desktop and mobile:**
![Screenshot from 2022-07-22 21-00-40](https://user-images.githubusercontent.com/80612925/180506255-0fa78acb-72c3-4af6-a26d-31f4a5f7548b.png)
![Screenshot from 2022-07-22 21-01-29](https://user-images.githubusercontent.com/80612925/180506315-9672baf5-14c6-4cc6-9386-a808514bb5a5.png)

**Figma popup:**
![Screenshot from 2022-07-22 21-05-01](https://user-images.githubusercontent.com/80612925/180506792-e45389e0-a955-4d38-830f-aaaa06be45f7.png)
![Screenshot from 2022-07-22 21-05-27](https://user-images.githubusercontent.com/80612925/180506819-b17dbcbf-9c8a-4ec1-9636-2e483033b187.png)